std_msgs/Header header
float32[] q_control_points
float32[] t_control_points
int32 num_q_control_points
int32 dim_q_control_points
int32 num_t_control_points
